### DISCUSSION TITLE
Ajout d'un contrôle retour 5secondes

### DISCUSSION THREAD

**Discussion by Koopa-13** [EsupPortail/Esup-Pod#411] [Ideas]
Bonjour, je voudrais vous soumettre l'idée d'un contrôle de retour en arrière sur les vidéos AMUpod de 5 secondes (ou 10 ou même 30), ou même avance de 5 secondes (ou 10 ou 30), car dès que la vidéo est un peu longue (1h et plus), une fois l'information manquée on est obligés de retourner minimum 45 secondes en arrière avec la souris.
En l'attente de votre réponse, je vous souhaite une bonne journée.

**Comment by ptitloup** [comment]
Bonjour, Nous avons également recu cette demande. Nous allons l'ajouter dans la roadmap de Pod pour l'ajouter des que possible.
Bonne journée, bien cordialement

**Reply by Koopa-13** [reply]
Merci beaucoup !
